### PR TITLE
Updating changelog for 10.5.0

### PR DESCRIPTION
**Pull request by ryanml**

### CODE REVIEW

**Review by danjm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,15 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Ensure ledger transport preference is set on import ([#12557](https://github.com/MetaMask/metamask-extension/pull/12557))
-- Don't show ledger live option in advanced settings if using firefox ([#12494](https://github.com/MetaMask/metamask-extension/pull/12494))
-- Turn off all old notifications ([#12500](https://github.com/MetaMask/metamask-extension/pull/12500))
-- Provide instructions for users switching from Ledger Live to WebHID ([#12519](https://github.com/MetaMask/metamask-extension/pull/12519))
-- Connect Ledger via WebHID ([#12411](https://github.com/MetaMask/metamask-extension/pull/12411))
+### Added
+- [#12411](https://github.com/MetaMask/metamask-extension/pull/12411): Add support for connecting Ledger devices to MetaMask via WebHID
+- [#12501](https://github.com/MetaMask/metamask-extension/pull/12501): Add "What's New" notification regarding Ledger WebHID support
+
+### Changed
```
Comment (Gudahtt) [inline CHANGELOG.md:14]: Nit: "Removed" might be more appropriate here

```suggestion
### Removed
```

**Review by Gudahtt — APPROVED** [review]
LGTM! One minor suggestion

File: CHANGELOG.md (hunk 1 above)
Comment (ryanml) [reply CHANGELOG.md:14]: I can change this, I just noticed we haven't used the `### Removed` header before so `### Changed` seemed the best for log consistency

Comment (ryanml) [reply CHANGELOG.md]: updated

**Pull request merged**